jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -70,6 +70,21 @@ public Builder addEntry(Path sourceFile, AbsoluteUnixPath pathInContainer) {
       return this;
```

Comment:
We could probably have just one constructor for `LayerEntry` and just call it ehre with like `null` for the third parameter (would need to use a permissions class).